home-application/src/main/java/com/softserveinc/ita/homeproject/application/config/JerseyConfig.java

```diff
@@ -36,6 +36,7 @@ public JerseyConfig() {
         // register endpoints
         ClassPathScanningCandidateComponentProvider scanner = new ClassPathScanningCandidateComponentProvider(false);
         property(ServletProperties.FILTER_STATIC_CONTENT_REGEX, ".*/apidocs/.*");
```

Comment:
I think this line can be removed as it is completely overridden by the next line.